src/fitnesse/resources/templates/accountPage.vm

```diff
@@ -0,0 +1,43 @@
+<div class="well">
+ <h2>Change Password</h2>
+ #if($request.getAuthorizationUsername())
+   <form method="post">
+    <input type="hidden" name="responder" value="saveAccount"/>
+     <fieldset>
```

Comment:
I think it is more secure, and customary to also require the current password to be required when changing ones password